core/src/main/java/nl/nn/adapterframework/extensions/graphviz/GraphvizEngine.java

```diff
@@ -157,7 +157,7 @@ private String getVisJsWrapper() {
 	private static class Env {
 		protected Logger log = LogUtil.getLogger(this);
 
-		J2V8 V8Instance = new J2V8();
+		JavascriptEngine<NashornScriptEngine> jsEngine = new Nashorn();
```

Comment:
This needs to be configurable. J2V8 is by far much faster then nashorn. Nashorn should be a backup if the v8 engine can't be loaded